src/Config/IConfigSchema.ts

```diff
@@ -0,0 +1,30 @@
+export interface IConfigSchema
+{
+
+	server: {
+		port: 3000;
```

Comment:
should be number

Reply:
fixed in 1fb8aa1f5e3f6f79acf87d923df8a14935bf086f